drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -31,6 +31,90 @@
 #define z_apb4_prescaler(v) LL_RCC_APB4_DIV_ ## v
 #define apb4_prescaler(v) z_apb4_prescaler(v)
 
+/* Macro to check for clock feasability */
+/* It is Cortex M7's responsibility to setup clock tree */
+/* This check should only be performed for the M7 core code */
+#ifdef CONFIG_CPU_CORTEX_M7
+/* Define primary oscillator frequencies */
+/* Suppress the cast to uint32_t which */
+/* prevents from compare with #if > operator*/
+/* original defines in stm32h7xx_hal_conf.h*/
+#define HSI_FREQ 64000000UL	/* HSI_VALUE ((uint32_t)64000000) */
+#define HSE_FREQ HSE_VALUE	/* HSE_VALUE overridden by the build system without cast to uint*/
+#define CSI_FREQ 4000000UL	/* CSI_VALUE ((uint32_t)4000000) */
+
+/* Choose PLL SRC */
+#ifdef CONFIG_CLOCK_STM32_PLL_SRC_HSI
+#define PLLSRC_FREQ  HSI_FREQ
+#elif defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI)
+#define PLLSRC_FREQ  CSI_FREQ
+#elif defined(CONFIG_CLOCK_STM32_PLL_SRC_HSE)
+#define PLLSRC_FREQ  HSE_FREQ
+#else
+#define PLLSRC_FREQ 0
+#endif
+
+/* Given source clock and dividers, computed the output frequency of PLLP */
+#define PLLP_FREQ(pllsrc_freq, divm, divn, divp)	(((pllsrc_freq)*(divn))/((divm)*(divp)))
```

Comment:
The line is over 80 characters. Please test like this (run the script from zephyr folder):
> ./scripts/checkpatch.pl -g HEAD-7

Following should work:
```
#define PLLP_FREQ(pllsrc_freq, divm, divn, divp)	(((pllsrc_freq)*\
							(divn))/((divm)*(divp)))
```

Reply:
For some reason, this still gives the 80 characters warning. Because of TAB I guess.

Reply:
I have done this change, lines are now less than 80 characters but checkpatch fails because tab space is set to 8 spaces instead of 4 tab spaces.

Reply:
> I have done this change, lines are now less than 80 characters but checkpatch fails because tab space is set to 8 spaces instead of 4 tab spaces.

I know you're working with VS Code. Please place following code in `Zephyr.code-workspace` file:
```
{
        "folders": [
        {
            "path": "."
        }
    ],
	"settings": {
		"files.associations": {
			"*.c": "c"
        },
        // The number of spaces a tab is equal to. This setting is overridden
        // based on the file contents when `editor.detectIndentation` is true.
        "editor.tabSize": 8,

        // Insert spaces when pressing Tab. This setting is overriden
        // based on the file contents when `editor.detectIndentation` is true.
        "editor.insertSpaces": false,

        // When opening a file, `editor.tabSize` and `editor.insertSpaces`
        // will be detected based on the file contents. Set to false to keep
        // the values you've explicitly set, above.
		"editor.detectIndentation": false,
		"editor.rulers": [80]
    }
}
```

Then place this file in `./zephyrproject` folder. It should give you proper settings. This file works fine for me and check patch runs properly.

Reply:
Thanks for the tip, so zephyr has always been working with 8 tab spaces ?

Reply:
AFAIK it is same as in linux kernel. Tab = 8 spaces. Check Linux kernel coding style, it is quite funny reading :)

Reply:
The indentation is now fixed 😄

Reply:
> Thanks for the tip, so zephyr has always been working with 8 tab spaces ?

Yes. It is.